### PR TITLE
Carrefour.com.ar

### PR DESCRIPTION
**Pull request by vlad555**
For #74

### CODE REVIEW

File: locations/spiders/dominos_pizza.py [hunk 1]
```diff
@@ -4,84 +4,86 @@
 import re
 
 from locations.items import GeojsonPointItem
-ALLOWED_ZIPS=[range(500,9899),range(10000,21299),range(21400,26899),range(27000,34299),range(21400,26899),range(27000,34299),
```
Comment (iandees) [inline locations/spiders/dominos_pizza.py]: Please don't include other spiders in this pull request.

**Comment by SelfnessAid** [comment]
@iandees 
I am getting error on this spider.
`import requests`
should I install this package on my side to test my spider?

**Comment by SelfnessAid** [comment]
screen shot 2017-12-12 at 9 42 02 am

**Comment by iandees** [comment]
There should be no reason to have the `requests` library. I don't see it included in this pull request.

**Comment by iandees** [comment]
Ah I see – I merged in a spider that has it included. It looks like you're trying to overwrite or replace the spider that already exists, though? If yours works without requests then please replace it.

**Comment by SelfnessAid** [comment]
I removed that file and committed my result.

**Comment by vlad555** [comment]
I'm sorry
You're right, 'requests' don't necessarily already.
 Sorry, didn't answer yesterday

**Comment by nicolas17** [comment]
Uh what? This replaced a spider giving 19 results with one giving 14 results? As I said, there are 157 stores in Buenos Aires province alone, so that can't be right.

**Comment by iandees** [comment]
@nicolas17 no need to be indignant. There's lots going on here.

I'll re-open the ticket this refers to.

**Comment by vlad555** [comment]
Sorry, my mistake.
Now I'm programming spider who use their search system thru cities database
Is this acceptable?

**Pull request merged**